cmd/buildah/manifest.go

```diff
@@ -221,6 +221,7 @@ func init() {
 	flags.StringVar(&manifestPushOpts.creds, "creds", "", "use `[username[:password]]` for accessing the registry")
 	flags.StringVar(&manifestPushOpts.digestfile, "digestfile", "", "after copying the image, write the digest of the resulting digest to the file")
 	flags.StringVarP(&manifestPushOpts.format, "format", "f", "", "manifest type (oci or v2s2) to attempt to use when pushing the manifest list (default is manifest type of source)")
+	flags.StringSliceVar(&manifestPushOpts.replicateWithCompression, "replicate", nil, "replicate instances with selected compression while pushing")
```

Comment:
I’m skeptical about the `replicate` name (`push --replicate=zstd` seems unintuitive to me), but, \*shrug\*, I can’t think of a good name.